Assignment/solution_student/index.html

```diff
@@ -1,35 +1,46 @@
+<!DOCTYPE>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>About me</title>
+		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Lora' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
+	</head>
+	<body>
+		<div class="nav1">
```

Comment:
Instead of `<div class="nav1">` you'd probably be better off using the html5 structural tag `<nav>` which is specifically for containing navigational elements. This way you wouldn't even need classes as as well because you could do something like this:
```
<header>
  <nav><!-- nav stuff --></nav>
</header>
<!-- main stuff -->
<footer>
  <nav><!-- nav stuff --></nav>
</footer>
``
And then target them specifically with nested selectors:
```
header nav { /* the styles */ }
footer nav { /* different styles */ }

We'll be going over nested selectors in the next lesson more